h2o-core/src/main/java/water/api/ModelMetricsHandler.java

```diff
@@ -367,7 +367,15 @@ public JobV3 predictAsync(int version, final ModelMetricsListSchemaV3 s) {
       if (parms._exemplar_index >= 0) {
         parms._predictions_name = "members_" + parms._model._key.toString() + "_for_exemplar_" + parms._exemplar_index;
       } else {
-        parms._predictions_name = "predictions" + Key.make().toString().substring(0, 5) + "_" + parms._model._key.toString() + "_on_" + parms._frame._key.toString();
+        switch (parms._model.getModelType()) {
```

Comment:
This seems to be the only usage of model type. Is this really worth complicating the API?

Reply:
Removed.